joomla/joomla-cms
# Extensions: Manage - Warning for not using Update

Pull Request to distinguish 3rd party extensions that do NOT support the Joomla Extension Update

### Summary of Changes
The JED lists for all extensions if they support the Joomla Extension Update.
However, you cannot see if the extensions you've installed in your Joomla site support the Joomla Extension Update. Maybe the JED lists some extensions that they support the Update, but you are using older versions in your site that do not support it.

To make it easier to see which extensions support the Joomla Extension Update, I've added a warning for 3rd party extensions that do not support it.

### Testing Instructions
Install a 3rd party extension that uses the Joomla Update, for example
https://db8.eu/download/component/db8-optimize-site

Install this patch.

Go to **Extensions: Manage**. You should see the installed 3rd party extension. 

![screen-1](https://cloud.githubusercontent.com/assets/1217850/26754076/ceeee2e2-4873-11e7-969f-e34586ee4560.png)


Go to **Extensions: Update Sites**. **Remove the reference** for the installed 3rd party extension.


![screen-2](https://cloud.githubusercontent.com/assets/1217850/26754078/cef07bac-4873-11e7-9b8a-7e159589a491.png)



![screen-3](https://cloud.githubusercontent.com/assets/1217850/26754077/ceef85a8-4873-11e7-9b8d-32dd3d59799a.png)

Go to **Extensions: Manage**. You should see the installed 3rd party extension plus a message that it does not support Joomla update

![screen-4](https://cloud.githubusercontent.com/assets/1217850/26754079/cef0b8ec-4873-11e7-9013-7dd6e40045b6.png)


The extension did support the Joomla Update, but the reference is missing in the Update Sites and not checked anymore. To enable it again: 
Go to **Extensions: Update Sites**. Choose Rebuild.

Go to **Extensions: Manage**. You should see the installed 3rd party extension without the warning message.
